### ISSUE TITLE
Drop support for `accelerate`

### ISSUE THREAD

**Issue by lkuper** [IntelLabs/ParallelAccelerator.jl#21]
None of our standard set of benchmarks use `accelerate` anymore.  ParallelAccelerator probably doesn't need to export it.

@DrTodd13 I notice that `embed` uses `accelerate`.  Is that necessary, or could it just use `@acc`?

**Comment by DrTodd13** [comment]
Yes, I think you can change accelerate in embed to @acc but I think you probably also then need to call the function with some small input size to actually force the ParallelAccelerator code to run. 

----- Original Message -----

**Comment by lkuper** [comment]
Apparently Domain IR also uses `accelerate`.  So we shouldn't drop it just yet.

**Comment by ninegua** [comment]
When DomainIR finds a function that needs to be translated, it will call accelerate, but what we really want is to apply the set of passes in OptFramework to do the translation instead of calling accelerate. I'll work on this.
